### DISCUSSION TITLE
Does the transformers in text/tabular predictors get fine-tuned?

### DISCUSSION THREAD

**Discussion by atabari-bci** [autogluon/autogluon#1803] [Q&A] [answered]
I was wondering if the base transformers used to generate embedding for text, in both text and tabular predictor, get fine-tuned during training (predictor.fit) or just used for embedding generation?

**Comment by Innixma** [comment, marked as answer]
They are fine-tuned. We don't pass text embeddings as features to pure tabular models, instead we use ngrams.

**Reply by atabari-bci** [reply]
How about tabular in "multimodal" mode? Doesn't it also include text predictors? Or the textPredictor alone. In general when ever transformers are used, are the base transformers being fine-tuned with my data or should I fine-tune it separately and add it to the model?

**Reply by Innixma** [reply]
Yes, if in "multimodal" mode the transformers / pre-trained models are always fine-tuned. You shouldn't fine-tune separately.
